server/src/routes.ts

```diff
@@ -2215,6 +2266,480 @@ export default async function routes(fastify: FastifyInstance) {
+              'mobile_image_url',
+              'mobile_image_link',
+              'footer_icon',
+              'secret_code'
+            ],
+            properties: {
+              name: { type: 'string' },
+              title_image: { type: 'string', format: 'binary' },
+              title_link: { type: 'string' },
+              header_link_text: { type: 'string' },
+              header_link_url: { type: 'string' },
+              header_color: { type: 'string' },
+              header_link_color: { type: 'string' },
+              main_color: { type: 'string' },
+              footer_color: { type: 'string' },
+              left_image_url: { type: 'string', format: 'binary' },
+              left_image_link: { type: 'string' },
+              right_image_url: { type: 'string', format: 'binary' },
+              right_image_link: { type: 'string' },
+              mobile_image_url: { type: 'string', format: 'binary' },
+              mobile_image_link: { type: 'string' },
+              footer_icon: { type: 'string', format: 'binary' },
+              secret_code: { type: 'number' },
+            },
+          },
+          response: {
+            200: {
+              type: 'object',
+              properties: {
+                id: { type: 'number' },
+                name: { type: 'string' },
+                title_image: { type: 'string' },
+                title_link: { type: 'string' },
+                header_link_text: { type: 'string' },
+                header_link_url: { type: 'string' },
+                header_color: { type: 'string' },
+                header_link_color: { type: 'string' },
+                main_color: { type: 'string' },
+                footer_color: { type: 'string' },
+                left_image_url: { type: 'string' },
+                left_image_link: { type: 'string' },
+                right_image_url: { type: 'string' },
+                right_image_link: { type: 'string' },
+                mobile_image_url: { type: 'string' },
+                mobile_image_link: { type: 'string' },
+                footer_icon: { type: 'string' },
+                secret_code: { type: 'number' },
+                created_date: { type: 'string' },
+                is_active: { type: 'boolean' },
+              },
+            }
+          },
+          security: [
+            {
+              "authorization": []
+            }
+          ]
+        },
+      },
+      async (req: any, res) => {
+        const title_image = req.body[Symbol.for('title_image')][0];
+        if (!title_image) {
+          return new createError.BadRequest('An image is required for field title_image');
+        }
+        if (title_image.mimetype != 'image/png') {
+          return new createError.BadRequest('title_image must be png');
+        }
+        const filename = 'title-image-' + (new Date().getTime()) + '.png'
+        const title_image_url = await uploadFile(filename, title_image.mimetype, title_image.data);
```

Comment:
I see that we are dropping the images in the Cloud Storage in the root folder. We talked about storing all of them in a folder.
I created in the bucket a folder named `templates/`. Please check if we can send all images to that folder please